build/yamls/antrea.yml

```diff
@@ -6920,7 +6920,7 @@ spec:
             mountPath: /var/run/antrea
       containers:
         - name: antrea-agent
-          image: "antrea/antrea-ubuntu:latest"
+          image: "antrea/antrea-agent-ubuntu:latest"
```

Comment:
Do you update this file manually? You need to update the `repository: "antrea/antrea-ubuntu"` to the right image, and configure it for agent and controller separately in the file `build/charts/antrea/values.yaml`. 
You may also update the `hack/generate-manifest.sh` to generate the right manifests automatically.

Reply:
You should also update prepare-asserts.sh to replace the manifest with the right new images.